### PR TITLE
Build doc

### PR DESCRIPTION
**Pull request by jerzerisz**
add a blurb regarding building locally/testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,4 +247,26 @@ virsh snapshot-create-as --domain vm_name --name snapshot_2
...
-```
\ No newline at end of file
+```
+
+### Development
+install libvirt-dev package:
+
+Debian:
+```shell
+apt-get install libvirt-dev
```
Comment (goneri) [inline README.md:257]: On Fedora, the package is called libvirt-devel. We also need python3-devel, and I suspect we need this one on Debian too to be able to prepare the venv.

**Review by goneri — COMMENTED** [review]
Thanks for the PR!

File: README.md [hunk 2]
```diff
@@ -231,3 +231,15 @@ networks:
...
     nic_model: e1000
 ```
+### Development
+
+You can run a development copy in a virtual env:
+
+```shell
+python3 -mvenv ~/tmp/my-greate-venv
+source ~/tmp/my-greate-venv/bin/activate
+pip3 install -e /path/to/my-virt-lightning-copy
```
Comment (BubaVV) [inline README.md:264]: AFAIR, you need pip3 install -U -e here to resolve possible conflict with VL installed system-wide

File: README.md (hunk 1 above)
Comment (BubaVV) [reply README.md:257]: https://github.com/virt-lightning/virt-lightning/commit/c47dac07ea8d1fb3cbefbf02e13bf9525cc21401 here is recently updated related document. Could you rebase your patch, or back-merge master to your branch to resolve?

Comment (jerzerisz) [reply README.md:257]: My bad, didn't test on nice clean easily created VM... luckily there is a tool for that.

Seems there are a few other common dependency's required, i'll track them down and update.

File: README.md (hunk 2 above)
Comment (jerzerisz) [reply README.md:264]: Total unfamiliar with pip, but reading a little, -U,--upgrade forces package upgrades, --user configures alternative install locations, which seems reasonable but it seems pip doesn't support --user in default venv setups

https://pip.pypa.io/en/stable/cli/pip_install/
https://pip.pypa.io/en/stable/user_guide/

I'll try a little testing, but so far i haven't noticed any conflict.

**Comment by goneri** [comment]
@jerzerisz, you need to rebase your branch before we can merge. If you don't feel super comfortable with git, I will be happy to do it for you.

Also, it would be great if you can squash all the commits in a single one.

**Comment by goneri** [comment]
Thank you @jerzerisz!

**Pull request merged**